tools/cldr-apps/src/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -519,7 +517,7 @@ private void processRequest(HttpServletRequest request, HttpServletResponse resp
                         mySession.userDidAction();
 
                         CLDRLocale locale = CLDRLocale.getInstance(loc);
-                        CheckCLDR.Options options = DataSection.getOptions(null, mySession, locale);
+                        CheckCLDR.Options options = DataSection.getOptions(null, mySession, locale); // TODO: why null for WebContext here?
```

Comment:
I think in this case, WebContext was only used for fetching the session.

Reply:
OK, it looks like a little refactoring might be appropriate but not very important. What's important for me is to clarify the dependencies of DataSection and the different ways it's constructed with getDataSection, DataSection.make having some args null and some not null for mysterious reasons.